src/lib.rs

```diff
@@ -188,14 +188,25 @@ impl PosixACL {
         PosixACL { acl }
     }
 
-    /// Read a file's ACL and return as `PosixACL` object.
+    /// Read a path's ACL and return as `PosixACL` object.
     pub fn read_acl(path: &Path) -> Result<PosixACL, SimpleError> {
+        Self::read_acl_flags(path, acl_sys::ACL_TYPE_ACCESS)
+    }
+
+    /// Read a path's default ACL and return as `PosixACL` object.
```

Comment:
Since this doesn't support files, how about "`Read a directory's default ACL ...`"

Reply:
Fixed